### PR TITLE
[Inference] Add support for FSMT model with bleu calculation example

### PR DESCRIPTION
**Pull request by xingyaoww**
Add support for [FSMT](https://huggingface.co/transformers/model_doc/fsmt.html) (Fairseq MachineTranslation) model
- Add scripts `hf_fsmt_export.py` for huggingface FSMT pretrained model export
- Add `test_inference_bleu.py` to verify bleu score on WMT19 de->en and en->de validation set
- Support for cases where encoder and decoder have different inner size
- Support for cases where Transformer does not has layer norm for embedding layer, now works for 3 cases: post-layernorm + has layernorm embedding, post-layernorm + no layernorm embedding, pre-layernorm + no layernorm embedding.

### CODE REVIEW

**Review by Taka152 — CHANGES_REQUESTED** [review]
better to move to tests rather than examples, because it is basically to test bleu
